src/ui/components/Location/component.ts

```diff
@@ -169,33 +158,65 @@ export default class LocationComponent extends Component {
   }
 
   async loadFromAPI(locationSignature: RecordSignature) {
-    let { store } = this.args;
+    let { store, localStore } = this.args;
     try {
-      let location: Location = await store.query((q) =>
+      this.location = await store.query((q) =>
         q.findRecord(locationSignature)
       );
-      let measurements: Measurement[] = await store.query((q) =>
+    } catch {
+      // Fail silently
+    }
+    try {
+      let transform = await localStore.pull((q) =>
```

Comment:
Should this be defined further down where it's used? I had a hard time understanding why this was needed…

Reply:
For some reason using `localStore` effectively makes an API request but this is a mistake. This should have been `remoteStore`. This helped me debug what Orbit's coordinator is doing and now I'm refining the strategies because some funny things are occurring in the store that affect this PR.